stats/stats.go

```diff
@@ -67,6 +70,22 @@ func ImportRegularly(store Store, interval time.Duration) {
 // library prefers.
 type Series map[time.Time]float64
 
+// MarshalJSON marshals a Series to the format expected by chart.js.
```

Comment:
Link to chart.js docs? https://www.chartjs.org/docs/latest/

Reply:
Sure!